engine/validation/fields.php

```diff
@@ -343,23 +343,87 @@ function render()
 class UploadField extends Field
 {
     public static $FILE_MIME = [
-        'jpg' => 'image/jpeg',
-        'png' => 'image/png',
-        'gif' => 'image/gif',
-        'doc' => 'application/msword',
-        'pdf' => 'application/pdf'
+        'image' => [
```

Comment:
Nice job with those annoying mime types 👌

This file is getting too big, we will need to split it up otherwise it's hard to maintain

Reply:
Maybe we can do this in another PR